app/static/uppy-test.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
This file should be moved to a template using the `.jinja2` format (see `app/templates/home.jinja2` for an example) and an accompanying route added to `app/modules/frontend/views_backend.py` to support this Houston-only test interface